README.md

```diff
@@ -0,0 +1,58 @@
+# arns-react
+A React App for the Arweave Name System Registry, which lets users search for and purchase Names.
+
+## Getting Started
+
+### Running the app locally
+
+- clone the repo
+- `npm install`
+- `npm run setup:contracts` - this will start arlocal in --persist mode, generate a wallet, and deploy the contracts
+- `npm run dev` - app will start on localhost:5173 and connect to arlocal.
+- open browser and navigate to localhost:5173
+The app should now be running, and you can connect your arconnect, arweave.app, or JWK wallet.
+
+
+
+## Testing
+
+### Running tests
+to test run `npm test`
+
+
+### What are we using?
+
+Testing required a bit of configuration to use Jest with vite. We are using:
+
+- jest \ runs our tests
+- @testing-library/react \ renders components in our test env
+- @testing-library/jest-dom \ asserts components are in the dom and contain data
+- @testing-library/user-event \ programmatically interacts with components
+- @babel/preset-react >
+- @babel/preset-typescript >
+- @babel/preset-env \ these babel libs allow use to use JSX, TS, and ES6 modules in tests 
+- identity-obj-proxy \ helps with css modules so we can see original class names
```

Comment:
nit: not really necessary - you can obtain this from the `package.json` and the README should be focused on how to use, not necessarily dependencies we port in

Reply:
fixed in commit c09a9a10ff8bee85bb707e9b9796cc3f134a0a88